ChatRPG/ChatRPG/Pages/Campaign.razor

```diff
@@ -0,0 +1,97 @@
+﻿@page "/Campaign"
+@using ChatRPG.Data
+@using ChatRPG.API
+@inject IConfiguration Configuration
+@inject IOpenAiLlmClient OpenAiLlmClient;
+@inject IJSRuntime JsRuntime
+
+<PageTitle>Campaign</PageTitle>
+
+<div class="text-center mx-auto">
+    <h1>Campaign</h1>
+</div>
+
+<div class="conversation mx-auto col-8 mt-4 rounded p-5 text-black">
+    <div class="conversation-text">
+        @foreach (string message in _conversation)
+        {
+            string prefix = message.StartsWith("Player: ") ? "user" : "assistant";
+            <div class="@($"{prefix}-message-container")">
+                <p class="@($"{prefix}-message")">@message</p>
+            </div>
+        }
+    </div>
+</div>
+
+<div class="container">
+    <div class="input-group mx-auto col-8 mt-2 mb-5 input-group-border">
+        <input type="text" style="resize:vertical;" class="form-control user-prompt custom-text-field"
+               @bind="_userInput" placeholder="What do you do?" @onkeyup="@EnterKeyHandler"/>
+        <div class="input-group-append">
+            <button class="btn btn-primary ml-2" type="button" @onclick="SendPrompt">
```

Comment:
Kan @ ikke fjernes fra EnterKeyHandler ligesom ved SendPrompt? Kan godt være den ikke kan, mindes der var varierende behavior i Blazor

Reply:
`EnterKeyHandler` skal have en `@` for at indikere at der er en binding - dvs. at den overfører parametre som har med eventen at gøre. `SendPrompt` tager ingen parametre og behøver derfor ikke en binding.